tgs/src/test/cpp/tgs/FeatureExtraction/CalculatorGenomeNodeTest.cpp

```diff
@@ -106,7 +106,7 @@ namespace Tgs
       }
 
       // testing equation ((3 * v) + sqrt(v)) - 2
-      SubtractCalculatorNode* root = new SubtractCalculatorNode();
+      boost::shared_ptr<SubtractCalculatorNode> root(new SubtractCalculatorNode());
```

Comment:
All of the objects below are added to a `std::map<std::string, boost::shared_ptr<CalculatorGenomeNode> >` object called `CalculatorGenomeNode::_inputs` so they are disposed of.  The `root` object is the only one that isn't cleaned up so I use a shared pointer to do that for me.